athletics.lic

```diff
@@ -234,11 +164,40 @@ class Athletics
     Flags.delete('climbing-finished')
     Flags.delete('climbing-too-easy')
     Flags.delete('climbing-too-hard')
+    Flags.delete('climbing-dead-rope')
     bput('stop climb', 'You stop practicing your climbing skills.', "You weren't practicing your climbing skills anyway.")
+    DRC.stop_playing
     fix_standing
     stow_hands
   end
 
+  def check_rope?
+    fput("study #{@settings.climbing_rope_adjective} rope")
```

Comment:
❓ Is it feasible to convert this to `bput`?

Reply:
Yeah, now I can.  Apparently the permanent ones have the same first line of study as the temp ones.